nss-day-cohort-44/nutshell-whiskered-gnomes
# Ks instructions

# Description
Add correct wireframe, dependency graph, and ERD files. Add planning dir in src. Delete images dir. Add INSTRUCTIONS.md and add markup with filepaths to the planning docs.

Fixes # (issue)
## Type of change
- [x] Document Change

# Testing Instructions
Click code tab in github repo
change branch to ks-instructions
navigate to src and click INSTRUCTIONS.md
check that the correct wireframe, dependency graph and ERD files show up on the page

# Checklist:
